local/dependencies.go

```diff
@@ -20,93 +20,225 @@ package local
 
```

Comment:
It's easier to look at the [final file](https://github.com/docker/compose-cli/blob/697964ca7cc0216e67ab4162bdaf4d849168b910/local/dependencies.go) rather than the diff